business-central-parent/business-central-distribution-wars/business-central/pom.xml

```diff
@@ -50,7 +50,7 @@
     <dependency>
       <groupId>org.kie</groupId>
       <artifactId>business-central-webapp</artifactId>
-      <classifier>thorntail</classifier>
+      <classifier>bootable</classifier>
```

Comment:
was this forgotten to apply? as I can see below it already uses standalone classifier?

Reply:
```suggestion
      <classifier>standalone</classifier>
```